read-before-readme.txt

```diff
@@ -0,0 +1,2 @@
+If permissions field added into action yml and it has no "actions:read" value, then we can not list
```

Comment:
What is the purpose of this file? Left over from Github action template?

Reply:
It is just reminder for permission error for actions that are run without actions:read permission. It has been removed.